lib/worker/index.js

```diff
@@ -39,13 +41,15 @@ Object.keys(queues).forEach(name => {
 
     Sentry.setTag('queue', name)
     Sentry.captureException(err)
+    statsd.increment('job-error')
   })
 
   queue.on('failed', async (job, err) => {
     app.log.error(`Error occurred while processing job id=${job.id} on queue name=${name}`)
 
     const subscription = await Subscription.getSingleInstallation(job.data.jiraHost, job.data.installationId)
     await subscription.update({ syncStatus: 'FAILED' })
+    statsd.increment('job-failed')
```

Comment:
I'm also wondering if there are any appropriate tags to pass across. But I'm not sure yet if there are.